doc/python/single_cell_model.rst

```diff
@@ -0,0 +1,74 @@
+.. _pysinglecellmodel:
+
+Single cell model
```

Comment:
This would be a good place to stress the single_cell_model is a Python exclusive. Maybe also the constraints (you cant run it distributed right? no domain decomp etc).